src/sagas/pushNotification.js

```diff
@@ -108,6 +146,7 @@ function* confirmDeviceRegistrationOnFirebase() {
 async function getDeviceId() {
   try {
     const deviceId = await messaging().getToken();
+    console.debug('DeviceID on FCM: ', '<add the deviceId here>');
```

Comment:
I don't like this log, not sure how sensitive FCM deviceId is, but this will show up on logcat in android, right? Maybe other apps have access to it, specially if the device is rooted

It's only helpful during development, so I suggest removing it

Reply:
Ok. I will add it to a patch.

Reply:
https://github.com/HathorNetwork/hathor-wallet-mobile/pull/268/commits/f23bac0296e563e0129150f3c87bfc0f9b936b79